registration-processor-default.properties

```diff
@@ -987,17 +989,51 @@ mosip.role.registration.getPostsync=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,RE
 mosip.role.registration.getPostsyncv2=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER,REGISTRATION_SUPERVISOR,RESIDENT
 auth.server.admin.allowed.audience=mosip-regproc-client,mosip-admin-client,mosip-resident-client,mosip-reg-client
 mosip.regproc.cbeff-validation.mandatory.modalities=Right,Left,Left RingFinger,Left LittleFinger,Right RingFinger,Left Thumb,Left IndexFinger,Right IndexFinger,Right LittleFinger,Right MiddleFinger,Left MiddleFinger,Right Thumb,EXCEPTION_PHOTO
-#updated properties name with the prefix (mosip.regproc)
+
+
 mosip.regproc.landing.zone.account.name=landing-zone
 mosip.regproc.landing.zone.type=ObjectStore
-
 mosip.regproc.landing.zone.fixed.delay.millisecs=120000
 mosip.regproc.landing.zone.inital.delay.millisecs=120000
 
 registration.processor.lostrid.max.registrationid=5
 mosip.registration.processor.lostrid.max-registration-date-filter-interval=30
+
 ##Ability to update contact information as part of LOST UIN
-uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress
+uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress,addressLine1,addressLine2,addressLine3
 
-##timeout in milliseconds for health check registrer
-mosip.regproc.health-check.handler-timeout=2000
```

Comment:
It should added with the same value in the master.

Reply:
Done